### PR TITLE
Mobile Service/get mobile os

### PR DESCRIPTION
**Pull request by oridanus**
to solve this - https://app.asana.com/0/1157695988834378/1160484435514194/f

### CODE REVIEW

File: projects/ui-framework/src/lib/services/utils/mobile.service.ts [hunk 1]
```diff
@@ -66,9 +74,29 @@ export class MobileService {
     return this.isMobBrowser;
   }
 
+  private getMobileOperatingSystem(): MobileOS {
```
Comment (galloween) [inline projects/ui-framework/src/lib/services/utils/mobile.service.ts]: what does it achieve? we can already detect touch devices and mobile browsers. why do we need to detect OS ?

File: projects/ui-framework/src/lib/services/utils/mobile.service.ts [hunk 2]
```diff
@@ -32,12 +38,14 @@ export class MobileService {
...
   isMobBrowser: boolean;
   isTouchDevice: boolean;
+  mobileOS: MobileOS;
 
   constructor(
     private windowRef: WindowRef,
     private utilsService: UtilsService
   ) {
     this.isMobBrowser = this.checkForMobileBrowser();
+    this.mobileOS = this.getMobileOperatingSystem();
```
Comment (galloween) [inline projects/ui-framework/src/lib/services/utils/mobile.service.ts]: lets add it to getMediaData() method as well

Comment (oridanus) [reply projects/ui-framework/src/lib/services/utils/mobile.service.ts]: added

File: projects/ui-framework/src/lib/services/utils/mobile.service.ts [hunk 3]
```diff
@@ -66,9 +74,29 @@ export class MobileService {
...
+      return MobileOS.windowsPhone;
+    }
+
+    if (/android/i.test(userAgent)) {
+      return MobileOS.android;
+    }
+
+    // iOS detection from: http://stackoverflow.com/a/9039885/177710
+    // @ts-ignore
+    if (/iPad|iPhone|iPod/.test(userAgent) && !window.MSStream) {
```
Comment (galloween) [inline projects/ui-framework/src/lib/services/utils/mobile.service.ts]: safer to test for lowercase + /i modifier (`/ipad|iphone|ipod/i`)

Comment (oridanus) [reply projects/ui-framework/src/lib/services/utils/mobile.service.ts]: fixed thanks!

**Pull request merged**